tests/libvmi/network.go

```diff
@@ -50,10 +50,10 @@ func InterfaceDeviceWithMasqueradeBinding(ports ...kvirtv1.Port) kvirtv1.Interfa
 	}
 }
 
-// InterfaceDeviceWithBridgeBinding returns an Interface named "default" with bridge binding.
-func InterfaceDeviceWithBridgeBinding() kvirtv1.Interface {
+// InterfaceDeviceWithBridgeBinding returns a named Interface with bridge binding.
```

Comment:
I think the `named` is not needed now. It just returns `an Interface`.

Reply:
Done